### PR TITLE
update documentation and reorganize directory structure

### PR DESCRIPTION
**Pull request by oryband**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Launch a Network
+
+The following are instructions on setting up a group of Core validators in order to setup a new or join an existing network.
+
+Kin Foundation are currently using AWS for managing its infrastructure, though other cloud providers are good as well.
```
Comment (doodyparizada) [inline INSTALL.md]: Kin Ecosystem

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+## Network Topology
+
+Before launching a group of nodes, it is important to decide on the following:
+
+1. How many nodes to run, keeping in mind high-availabiltiy considerations:
+    1. We recommend the network size include at least a total 7 nodes, to allow for more than one node to fail.
```
Comment (doodyparizada) [inline INSTALL.md]: network to be comprised of at least 7 nodes

File: INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    1. Nodes should be deployed on different cloud regions e.g. AWS us-east-1 and us-west-1. Prefer a separate region for each node.
+1. Decide on your [quorum set and safety threshold percentage](https://www.stellar.org/developers/stellar-core/software/admin.html#crafting-a-quorum-set).
+
+## Core
+
+1. [Configure Core](https://www.stellar.org/developers/stellar-core/software/admin.html).
+    - See Stellar's [configuration example](https://github.com/stellar/stellar-core/blob/master/docs/stellar-core_example.cfg).
+    - See [our example](deploy/ansible/playbooks/roles/stellar-core/templates/stellar-core.cfg.j2).
+1. See how to [install and run Core](https://github.com/stellar/stellar-core/blob/master/INSTALL.md).
+    - Kin Foundation runs Core using Docker.
```
Comment (doodyparizada) [inline INSTALL.md]: Ecosystem

File: INSTALL.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+1. Launch an EC2 and RDS instance, similar to the above.
+1. [Configure Horizon](https://www.stellar.org/developers/horizon/reference/admin.html#configuring)
+    - Similar to Core, we run Horizon using Docker.
+    - See [images/Dockerfile.Horizon](images/Dockerfile.horizon) and [images/docker-compose.yml](images/docker-compose.yml)
+as an example on how to build and configure Horizon on Ubuntu 16.04.
+1. Finally, see Horizon's [Administration Guide](https://www.stellar.org/developers/horizon/reference/admin.html).
+
+## Automation
+
+At Kin we use Ansible for orchestration and Terraform for infrastructure automation.
```
Comment (doodyparizada) [inline INSTALL.md]: Kin Ecosystem

File: INSTALL.md (hunk 1 above)
Comment (oryband) [reply INSTALL.md]: I'm referring to the Kin NPO, the "ecosystem" can be confusing in the case and might be misinterpreted to the entire ecosystem.

**Pull request merged**